### PR TITLE
Add `sys.node_checks` update support

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

File: sql/src/main/java/io/crate/analyze/UpdateStatementAnalyzer.java [hunk 1]
```diff
@@ -102,31 +102,38 @@ public AnalyzedStatement visitUpdate(Update node, Analysis analysis) {
             throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
                     "relation \"%s\" doesn't support update operations", analyzedRelation));
         }
-        assert analyzedRelation instanceof DocTableRelation : "sourceRelation must be a DocTableRelation";
-        DocTableRelation tableRelation = ((DocTableRelation) analyzedRelation);
 
+        FieldResolver fieldResolver = (FieldResolver) analyzedRelation;
```
Comment (seut) [inline sql/src/main/java/io/crate/analyze/UpdateStatementAnalyzer.java]: afaik we should at least add an assertion to check for a FieldResolver instance

File: sql/src/test/java/io/crate/integrationtests/SysNodeCheckerIntegrationTest.java [hunk 2]
```diff
@@ -43,4 +44,14 @@ public void testChecksPresenceAndSeverityLevels() throws Exception {
         assertThat((Integer) response.rows()[3][0], is(SysCheck.Severity.MEDIUM.value()));
     }
 
+    @Test
+    public void testUpdateAcknowledge() throws Exception {
+        SQLResponse resp = execute("select id, passed from sys.node_checks where passed = false");
```
Comment (seut) [inline sql/src/test/java/io/crate/integrationtests/SysNodeCheckerIntegrationTest.java]: what check did fail here and is false, don't get it ;) is this deterministic?

**Comment by aelesbao** [comment]
ack

**Comment by seut** [comment]
lgtm

**Pull request merged**